Snippets/Templates/Templates_3/NServiceBusDockerContainer.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Logging;
+
+namespace Templates_2
```

Comment:
namespace doesn't match, should be `Templates_3`

Reply:
Forgot to update those. Will do.